### PR TITLE
Add FlowDirection to IFrameworkElement

### PR DESCRIPTION
**Pull request by hartez**
Implement RTL layout for HorizontalStackLayout
Make LayoutExtensions RTL-aware RE: margins

### CODE REVIEW

**Review by Redth — APPROVED** [review]

File: src/Core/src/Core/FlowDirection.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Xamarin.Platform
```
Comment (rmarinho) [inline src/Core/src/Core/FlowDirection.cs:1]: Wrong namespace

File: src/Core/src/Primitives/FlowDirection.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Maui
+{
+	public enum FlowDirection
```
Comment (rmarinho) [inline src/Core/src/Primitives/FlowDirection.cs:3]: don't we have this above ?

File: src/Core/src/Core/FlowDirection.cs (hunk 1 above)
Comment (hartez) [reply src/Core/src/Core/FlowDirection.cs]: Argh, repo move error. Fixed.

File: src/Core/src/Primitives/FlowDirection.cs (hunk 2 above)
Comment (hartez) [reply src/Core/src/Primitives/FlowDirection.cs:3]: Yeah, the one from the Forms repo snuck across. Fixed.

**Review by rmarinho — APPROVED** [review]

**Pull request merged**